hubitatmaker/const.py

```diff
@@ -92,3 +97,5 @@
 STATE_UNKNOWN = "unknown"
 STATE_UNLOCKED = "unlocked"
 STATE_UNLOCKED_WITH_TIMEOUT = "unlocked with timeout"
+
+DEFAULT_SPEEDS = ["low", "medium-low", "medium", "medium-high", "high", "on", "off", "auto"]
```

Comment:
Could we call this `DEFAULT_FAN_SPEEDS` for future-proofing?

Reply:
Absolutely